motor/discovery/aws/mql_asset_objects.go

```diff
@@ -0,0 +1,283 @@
+package aws
+
+import (
+	"strings"
+
+	"github.com/cockroachdb/errors"
+	"go.mondoo.com/cnquery/motor/asset"
+	"go.mondoo.com/cnquery/motor/providers"
+)
+
+func getTitleFamily(awsObject awsObject) (awsObjectPlatformInfo, error) {
+	switch awsObject.service {
+	case "s3":
+		if awsObject.objectType == "bucket" {
+			return awsObjectPlatformInfo{title: "AWS S3 Bucket", platform: "aws.s3_bucket"}, nil
```

Comment:
We should align on the platform format. In k8s we have defined those as `k8s-pod`, `k8s-deployment`, etc. If we want to be consistent I guess we should used `aws-s3-bucket`?

Reply:
sounds good to me!

Reply:
i also renamed the core platform to aws-core in one of the commits, need to undo that back to just "aws". mostly just noting that here so i remember